### PR TITLE
Add requirements

### PR DESCRIPTION
**Pull request by butenkor**
@mht-dotsource @panshul007 Please review

### CODE REVIEW

File: doc/REQUIREMENTS.MD [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+Example configuration 2:
+ ```
+ {
+   "ctp": {
+     "client_secret": "xxx",
+     "client_id": "xxx",
+     "project_key": "xxx"
+   },
+   "emailSenderClassPath": "io.commercetools.sunrise.email.smtp.SmtpAuthEmailSender",
```
Comment (schleichardt) [inline doc/REQUIREMENTS.MD]: for the keys camelCase and snake_case are mixed (see client_secret)

File: doc/REQUIREMENTS.MD [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  "emailSenderUrl": "https://mydomain.com/sendorderconfirmation/order-uuid",
+  "email-send-channel-key": "order-confirmation-email",
+  "email-error-channel-key": "order-confirmation-email-error"
+}
+```
+
+Example configuration 2:
+ ```
+ {
+   "ctp": {
```
Comment (schleichardt) [inline doc/REQUIREMENTS.MD]: api and auth url are missing, see also the config for the Sunrise data import: https://github.com/commercetools/commercetools-sunrise-data/tree/master/importApp#prepare-a-payload-json-file

File: doc/REQUIREMENTS.MD [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+     "port": "xxx",
+     "transportSecurity": "xxx",
+     "username": "xxx",
+     "password": "xxx"
+   },
+   "emailSenderTemplate": {
+     "custome-object-container": "email",
+     "custom-object-key": "order-confirmation-template"
+   }
+   "email-send-channel-key": "order-confirmation-email",
```
Comment (schleichardt) [inline doc/REQUIREMENTS.MD]: now keys with dashes?

File: doc/REQUIREMENTS.MD (hunk 1 above)
Comment (butenkor) [inline doc/REQUIREMENTS.MD]: @schleichardt right, thanks: https://github.com/commercetools/commercetools-order-message-processor/commit/6ff1563e43b0846f5bbde823871848dd39ce85be

File: doc/REQUIREMENTS.MD [hunk 4]
```diff
@@ -18,43 +18,49 @@ Default is to fetch orders from last 5 days (5d). It should process configured a
...
+       "customObjectKey": "orderConfirmationTemplate"
+     }
    },
-   "emailSenderTemplate": {
-     "custome-object-container": "email",
-     "custom-object-key": "order-confirmation-template"
-   }
-   "email-send-channel-key": "order-confirmation-email",
-   "email-error-channel-key": "order-confirmation-email-error"
+   "emailSentChannelKey": "orderConfirmationEmail",
```
Comment (mht-dotsource) [inline doc/REQUIREMENTS.MD:85]: @butenkor adjust spelling to "send"

**Comment by butenkor** [comment]
@mht-dotsource @panshul007 please merge if ok

**Pull request merged**